cabotage/server/models/projects.py

```diff
@@ -84,7 +84,7 @@
 
 
 class Project(db.Model, Timestamp):
-    __versioned__ = {}
+    _versioned__: dict = {}
```

Comment:
all of the matching changes in this file remove the leading `_` which breaks sqlalchemy-continuum:

```
cabotage-app-1         | Traceback (most recent call last):
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/gunicorn/workers/gthread.py", line 282, in handle
cabotage-app-1         |     keepalive = self.handle_request(req, conn)
cabotage-app-1         |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/gunicorn/workers/gthread.py", line 334, in handle_request
cabotage-app-1         |     respiter = self.wsgi(environ, resp.start_response)
cabotage-app-1         |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/sentry_sdk/integrations/flask.py", line 84, in sentry_patched_wsgi_app
cabotage-app-1         |     return SentryWsgiMiddleware(lambda *a, **kw: old_app(self, *a, **kw))(
cabotage-app-1         |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/sentry_sdk/integrations/wsgi.py", line 109, in __call__
cabotage-app-1         |     reraise(*_capture_exception())
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/sentry_sdk/utils.py", line 1640, in reraise
cabotage-app-1         |     raise value
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/sentry_sdk/integrations/wsgi.py", line 102, in __call__
cabotage-app-1         |     response = self.app(
cabotage-app-1         |                ^^^^^^^^^
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/sentry_sdk/integrations/flask.py", line 84, in <lambda>
cabotage-app-1         |     return SentryWsgiMiddleware(lambda *a, **kw: old_app(self, *a, **kw))(
cabotage-app-1         |                                                  ^^^^^^^^^^^^^^^^^^^^^^^
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/flask/app.py", line 2213, in __call__
cabotage-app-1         |     return self.wsgi_app(environ, start_response)
cabotage-app-1         |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/flask/app.py", line 2193, in wsgi_app
cabotage-app-1         |     response = self.handle_exception(e)
cabotage-app-1         |                ^^^^^^^^^^^^^^^^^^^^^^^^
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/flask/app.py", line 2190, in wsgi_app
cabotage-app-1         |     response = self.full_dispatch_request()
cabotage-app-1         |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/flask/app.py", line 1486, in full_dispatch_request
cabotage-app-1         |     rv = self.handle_user_exception(e)
cabotage-app-1         |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/flask/app.py", line 1484, in full_dispatch_request
cabotage-app-1         |     rv = self.dispatch_request()
cabotage-app-1         |          ^^^^^^^^^^^^^^^^^^^^^^^
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/flask_debugtoolbar/__init__.py", line 157, in dispatch_request
cabotage-app-1         |     return view_func(**req.view_args)
cabotage-app-1         |            ^^^^^^^^^^^^^^^^^^^^^^^^^^
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/flask_login/utils.py", line 290, in decorated_view
cabotage-app-1         |     return current_app.ensure_sync(func)(*args, **kwargs)
cabotage-app-1         |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cabotage-app-1         |   File "/opt/cabotage-app/src/cabotage/server/user/views.py", line 288, in project_application
cabotage-app-1         |     view_releases=version_class(Release)
cabotage-app-1         |                   ^^^^^^^^^^^^^^^^^^^^^^
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/sqlalchemy_continuum/utils.py", line 123, in version_class
cabotage-app-1         |     manager = get_versioning_manager(model)
cabotage-app-1         |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cabotage-app-1         |   File "/opt/cabotage-app/lib/python3.11/site-packages/sqlalchemy_continuum/utils.py", line 30, in get_versioning_manager
cabotage-app-1         |     raise ClassNotVersioned(cls.__name__)
cabotage-app-1         | sqlalchemy_continuum.exc.ClassNotVersioned: Release
```